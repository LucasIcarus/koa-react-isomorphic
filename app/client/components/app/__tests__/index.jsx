import React from "react";
import { shallow } from "enzyme";
import App from "..";

test("App in development mode", () => {
  const NODE_ENV = process.env.NODE_ENV;
  process.env.NODE_ENV = "development";

  expect(
    shallow(<App routes="New Routes" />)
  ).toMatchSnapshot();

  process.env.NODE_ENV = NODE_ENV;
});

test("App in production mode", () => {
  expect(
    shallow(<App routes="New Routes" />)
  ).toMatchSnapshot();
});
