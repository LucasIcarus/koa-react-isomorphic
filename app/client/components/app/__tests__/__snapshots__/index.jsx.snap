// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App in development mode 1`] = `
<AppContainer>
  <div />
</AppContainer>
`;

exports[`App in production mode 1`] = `<div />`;
